en/deployment/articles/01-deploying-liferay/04-installing-liferay.markdown

```diff
@@ -81,6 +83,9 @@ For the administrator, supply the following information:
 
 **Email:** the default administrator user's email address
 
+| **Note:** the domain of administrator user's email will be used as the default
+| domain name of the instance (company.default.web.id configuration property).
```

Comment:
@jorgediaz-lr  - By company.default.web.id configuration property, do you mean the [`company.default.web.id` *portal property*](http://docs.liferay.com/ce/portal/7.2-latest/propertiesdoc/portal.properties.html#Company)?

Reply:
Hi @jhinkey , yes it is that `portal.properties` property